tools/spec-aggregator/main.go

```diff
@@ -12,28 +13,32 @@ import (
 	log "github.com/sirupsen/logrus"
 )
 
-var commonSpecPath = "/api3/common/openapi.yaml"
-var coreSpecPath = "/api3/core/openapi.yaml"
-var notificationStubPath = "/api3/notifications/openapi.stub.yaml"
-var notificationSpecPath = "/api3/notifications/openapi.gen.yaml"
-var globalSpecPath = "/api3/global/openapi.gen.yaml"
-var modulesPath = "/backend/pkg/modules/internal"
-var inModulePath = "restapi/openapi.yaml"
-var rootPath = "../.."
+const (
+	commonSpecPath         = "/api3/common/openapi.yaml"
+	coreSpecPath           = "/api3/core/openapi.yaml"
+	notificationStubPath   = "/api3/notifications/openapi.stub.yaml"
+	notificationSpecPath   = "/api3/notifications/openapi.gen.yaml"
+	globalSpecPath         = "/api3/global/openapi.gen.yaml"
+	modulesPath            = "/backend/pkg/modules/internal"
+	inModulePath           = "restapi/openapi.yaml"
+	rootPath               = "../.."
+	parentNotificationName = "APIClarityNotification"
+	baseNotificationName   = "BaseNotification"
+)
 
 func loadSpec(path string, fail bool) *openapi3.T {
 	loader := openapi3.NewLoader()
 	loader.IsExternalRefsAllowed = true
 
 	spec, err := loader.LoadFromFile(path)
 	if err != nil {
-		log.Errorf("Unable to load spec %s: %v", path, err)
+		log.Warnf("Unable to load spec %s: %v", path, err)
```

Comment:
if we don't want to fail so no need to print it as a warning. we can just put the message in the `log.Fatal`